src/node/nodetonode.h

```diff
@@ -49,6 +49,27 @@ namespace ccf
       channels = std::make_unique<ChannelManager>(network_pkey);
     }
 
+    template <class T>
+    void send(const NodeMsgType& msg_type, NodeId to, const T& data)
+    {
+      auto& n2n_channel = channels->get(to);
+      if (n2n_channel.get_status() != ChannelStatus::ESTABLISHED)
+      {
+        established_channel(to);
+        return;
+      }
+
+      to_host->write(node_outbound, to, msg_type, data);
+    }
+
+    template <class T>
+    const T& recv(const uint8_t*& data, size_t& size)
+    {
+      const auto& t = serialized::overlay<T>(data, size);
+      auto& n2n_channel = channels->get(t.from_node);
```

Comment:
This assignment isn't used. If it's safe to call `get` later when the created Channel will be used, we can just remove this line. If we need to ensure it is in the channels map _here_, we should add a comment explaining why we're calling `get`, but can remove the assignment.